CONTRIBUTING.md

```diff
@@ -0,0 +1,100 @@
+# Contributing
+
+### 1. Setting up a pre-commit Hook
+
+In the root directory, run the following commands:
+
+```
+cp .githooks/pre-commit .git/hooks/pre-commit
+chmod +x .git/hooks/pre-commit
+```
+
+Eslint is used to check the FE code, and [python black](https://black.readthedocs.io/en/stable/) is utilized to check the BE code.
+
+To ensure that the eslint pre-commit check is running correctly, create a branch with a commit that contains a `console.log`. If the commit is unsuccessful with the message: `Fix eslint errors and try again`, then the pre-commit is working correctly.
+
```

Comment:
Need to update the documentation to include the possibility that python black errors need to be fixed.

Reply:
Fixed in df602a71b980d6573ae7050c2cdb0dd0289ab17c